src/main.rs

```diff
@@ -63,3 +57,21 @@ fn main() {
 
     run(&args, &mut client);
 }
+fn handle_config_subcommand(client: &mut chatgpt::GptClient, config_sc: &ConfigSubCommand) {
+    if let Some(ref set) = config_sc.set {
+        let parts: Vec<&str> = set.split('=').collect();
+        if parts.len() == 2 {
+            client.set_config_value(parts[0], parts[1]);
+            println!(
+                "Configuration set successfully for {} to {}",
+                parts[0], parts[1]
+            )
+        } else {
+            println!("Invalid format for setting configuration. Use key=value");
```

Comment:
♻️ With relation to my previous comment, I think we should print an example here.